lib/ui/settings/settings.dart

```diff
@@ -61,12 +61,18 @@ class _SettingsState extends State<Settings> {
               MergeSemantics(
                 child: ListTile(
                   shape: const RoundedRectangleBorder(side: BorderSide.none),
-                  title: Text(L.of(context)!.settings_theme_switch_label),
-                  trailing: Switch.adaptive(
-                      value: snapshot.data!.theme == 'dark',
-                      onChanged: (value) {
-                        settingsBloc.darkMode(value);
-                      }),
+                  title: Text(L.of(context)!.settings_theme_dropdown_label),
+                  trailing: DropdownMenu<String>(
+                      initialSelection: snapshot.data!.theme,
+                      onSelected: (value) {
+                        settingsBloc.themeMode(value!);
+                      },
+                      dropdownMenuEntries: [
```

Comment:
If you set the theme to Auto and restart Anytime, when you go back into settings rather than Auto being selected either Light or Dark will be selected depending upon the current theme.

Reply:
Yes, as behaviour is abnormal. I fix it.